notebook/static/tree/js/notebooklist.js

```diff
@@ -595,6 +597,20 @@ define([
             $('.delete-button').css('display', 'none');
         }
 
+        // View is visible when an item is renderable or downloadable
+        if (selected.length > 0 && !has_directory) {
```

Comment:
Ok, then I believe the expected behavior for "View" is to open notebooks as a notebook vs. raw file (see my screencap below for an example). Given that, we can remove "View" entirely because viewing and editing any file other than a notebook is the same (e.g. open it in a text editor) and editing a notebook is to open it.